SEPRTR/iocBoot/iocSEPRTR-IOC-01/config.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+<config_part>
+	<ioc_desc>Muon Separator PSU</ioc_desc>
+	<ioc_details>Detailed status of PSU</ioc_details>
+	<macros>
+		<macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM port" />
+	</macros>
```

Comment:
Nit: I prefer the use of four spaces over tabs for compatability between text editors